attestation-service/src/bin/restful-as.rs

```diff
@@ -114,8 +115,39 @@ fn configure_cors(allowed_origin: &[String]) -> Cors {
 
 #[actix_web::main]
 async fn main() -> Result<(), RestfulError> {
-    env_logger::init_from_env(env_logger::Env::new().default_filter_or("info"));
-
+    let filter = EnvFilter::try_from_default_env()
+        .unwrap_or(EnvFilter::new("info"))
+        .add_directive("hyper=warn".parse().unwrap())
+        .add_directive("tonic=warn".parse().unwrap())
+        .add_directive("h2=warn".parse().unwrap())
+        .add_directive("tokio=warn".parse().unwrap())
+        .add_directive("mio=warn".parse().unwrap())
+        .add_directive("sled=warn".parse().unwrap());
+    tracing_subscriber::fmt()
+        .with_env_filter(filter)
+        .with_level(true)
+        .init();
+
+    let version = format!(
+        r"
+ ________  ________  ________  ________  ________  ________      
+|\   ____\|\   __  \|\   ____\|\   __  \|\   __  \|\   ____\     
+\ \  \___|\ \  \|\  \ \  \___|\ \  \|\  \ \  \|\  \ \  \___|_    
+ \ \  \    \ \  \\\  \ \  \    \ \  \\\  \ \   __  \ \_____  \   
+  \ \  \____\ \  \\\  \ \  \____\ \  \\\  \ \  \ \  \|____|\  \  
+   \ \_______\ \_______\ \_______\ \_______\ \__\ \__\____\_\  \ 
+    \|_______|\|_______|\|_______|\|_______|\|__|\|__|\_________\
+                                                     \|_________|
+                                                                                    
+version: v{}
+commit: {}
+buildtime: {}",
+        build::PKG_VERSION,
+        build::COMMIT_HASH,
+        build::BUILD_TIME
+    );
+
+    info!("Welcome to Confidential Containers Attestation Service!\n{version}");
```

Comment:
Maybe we should mention that this is the restful mode here (and similar on the other ones)